libcontainer/specconv/spec_linux.go

```diff
@@ -492,6 +493,12 @@ func CreateLibcontainerConfig(opts *CreateOpts) (*configs.Config, error) {
 			}
 		}
 	}
+
+	// update timens offsets
+	if spec.Linux.TimeOffsets != nil {
```

Comment:
The error in `CI` is in here. Please move this to the scope of the condition check in L384.

Reply:
L384: `if spec.Linux != nil {`

Reply:
Thank you for the suggestion @lifubang , I'll try it.

Reply:
I agree and moved the check `spec.Linux.TimeOffsets` into `spec.Linux`.
But the CI was failing due to logrus print statement added by mistake.
The CI is passing now